tajo-catalog/tajo-catalog-server/src/main/java/org/apache/tajo/catalog/store/AbstractDBStore.java

```diff
@@ -1023,6 +1026,77 @@ public void alterTable(CatalogProtos.AlterTableDescProto alterTableDescProto) th
 
   }
 
+  private Map<String, String> getTableOptions(final int tableId) throws CatalogException {
+    Connection conn = null;
+    PreparedStatement pstmt = null;
+    ResultSet res = null;
+    Map<String, String> options = TUtil.newHashMap();
+
+    try {
+      String tidSql = "SELECT key_, value_ FROM " + TB_OPTIONS + " WHERE TID=?";
+
+      conn = getConnection();
+      pstmt = conn.prepareStatement(tidSql);
+      pstmt.setInt(1, tableId);
+      res = pstmt.executeQuery();
+
+      while (res.next()) {
+        String key = res.getString("KEY_");
+        String value = res.getString("VALUE_");
+        options.put(key, value);
```

Comment:
New objects looks like unnecessary resources. I would like t more better if you don't create new String objects as follows.
```
options.put(res.getString("KEY_"), res.getString("VALUE_"));
```